ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { StudyLanguage } from "../models/studyLanguage";
+import { LanguageServiceAgent } from "../services/serviceAgents/languageServiceAgent.service";
+import { isNullOrUndefined } from "util";
+import { TranslateService } from "@ngx-translate/core";
+
+@Component({
+  selector: 'ddp-language-selector',
+  template: `
+    <div [hidden]="!loaded">
+      <button class="SimpleButton" [ngClass]="{'SimpleButton--Scrolled': isScrolled}" *ngIf="currentLanguage !== null && currentLanguage !== undefined" [matMenuTriggerFor]="menu">
```

Comment:
Can we use `isNullOrUndefined` function here?
```suggestion
      <button class="SimpleButton" [ngClass]="{'SimpleButton--Scrolled': isScrolled}" *ngIf="!isNullOrUndefined(currentLanguage)" [matMenuTriggerFor]="menu">
```

Reply:
Turns out Angular doesn't like that for some reason, but I found a different place where I was checking manually rather than calling the method and changed that location at least.